payroll/models/hr_payslip.py

```diff
@@ -320,6 +325,15 @@ def unlink(self):
             )
         return super(HrPayslip, self).unlink()
 
+    def compute_payslip_data(self):
+        """
+        Triggers a recompute of the worked_day_lines and other_inputs data.
+        This is useful if you need to manually recompute the values fetched from other modules
+        and don't want to trigger onchage manually.
+        """
+        for payslip in self:
+            payslip.onchange_employee()
+
```

Comment:
If all you're doing is calling onchange_employee() then why not just get rid of compute_payslip_data() and just call onchange_employee() directly?

Reply:
You are right, will change it next commit.